src/tools/rados/rados.cc

```diff
@@ -144,7 +144,7 @@ void usage(ostream& out)
 "       List all advisory locks on an object\n"
 "   lock get <obj-name> <lock-name>\n"
 "       Try to acquire a lock\n"
-"   lock break <obj-name> <lock-name> <locker-name>\n"
+"   lock break <obj-name> <lock-name> <locker-name> --lock-cookie <locker cookie>\n"
```

Comment:
i think "get" command requires "lock-cookie", and "lock-duration", "lock-description", "lock-type" are optional. could you help update "lock get" command with these options as well?

Reply:
```
[root@c165 ~]# rados -p cephfs_data lock info test5.0000000000000000 striper.lock
{"name":"striper.lock","type":"exclusive","tag":"","lockers":[]}[root@c165 ~]# 

[root@c165 ~]# rados -p cephfs_data lock get test5.0000000000000000 striper.lock
 
[root@c165 ~]# rados -p cephfs_data lock info test5.0000000000000000 striper.lock
{"name":"striper.lock","type":"exclusive","tag":"","lockers":[{"name":"client.4250","cookie":"","description":"","expiration":"0.000000","addr":"x.x.x.x:0/1677787373"}]}
 
[root@c165 ~]# rados -p cephfs_data lock break test5.0000000000000000 striper.lock client.4250 --locker-cookie ""
```

@tchaikov  From above tests, "get" command didn't require those "lock-XXX" options? If user set those options when getting lock, it required "locker-cookie" additionally only when breaking lock.

Reply:
@tchaikov ping:-)

Reply:
@david-z could you double check following code?

https://github.com/ceph/ceph/blob/05d685dd37b34f2a015e77124c537f3f8e663152/src/tools/rados/rados.cc#L1276-L1281

Reply:
"lock-cookie" is not "required" when getting the lock. but since you are updating the help message of "break" subcommand to document "lock-cookie". one might wonder where the lock-cookie comes from in the first place. so i am wondering if we should document this option in the help message or in the manpage.